src/components/base/BaseSelect.vue

```diff
@@ -0,0 +1,59 @@
+<script setup lang="ts">
+interface Props {
+  options: string[];
+  modelValue: string;
+}
+const props = withDefaults(defineProps<Props>(), {
+  options: null,
+  modelValue: "",
+});
+</script>
+
+<template>
+  <div class="base-dropdown">
+    <label class="case-dropdown-__label" for=""><slot /></label>
+    <select
+      class="base-dropdown__select"
+      :modelValue="props.modelValue"
+      @change="
+        $emit('update:modelValue', ($event.target as HTMLInputElement).value)
+      "
+    >
+      <option
+        v-for="option in props.options"
+        :key="option"
+        class="base-dropdown__option"
+        :value="option"
+      >
+        {{ option }}
+      </option>
+    </select>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+.base-dropdown {
+  &__label {
+    font-family: Roboto, Mono;
```

Comment:
If you want to use the same font on all elements it would maybe be good to update main.scss and set font family in all elements block

```
* { ... }
```